audioread/ffdec.py

```diff
@@ -93,6 +93,21 @@ def popen_multiple(commands, command_args, *args, **kwargs):
                 raise
 
 
+def available():
+    """Detect if the FFmpeg backend can be used on this system."""
+    proc = popen_multiple(
+        COMMANDS,
+        ['-version'],
+        stdout=subprocess.PIPE,
+        stderr=subprocess.PIPE,
+    )
+    proc.wait()
+    if proc.returncode == 0:
+        return True
+    else:
+        return False
```

Comment:
Just `return proc.returncode == 0` or `return not bool(proc.returncode)` would be a little more concise than this `if`/`else`.

Reply:
Agreed